Synthetic Data Generation/README.md

```diff
@@ -78,5 +146,27 @@ proc datasets library=sampsio; run;
 
 ## Change Log
 
-Version 1.0 (06OCT2022)
-
+- Version 1.1 (16AUG2023):
```

Comment:
This list does not render properly caused by

- a missing minus (-) sign in front of the subsection headers.
- a mixture of using tabs and spaces for indenting, although I'm not 100% sure that will cause rendering issues. Still use one approach only, and as there are only a few tabs in this file I suggest to replace the tab-character with spaces. 

So the lines for the 16AUG update would roughly read like this:

- Version 1.1 (16AUG2023):
  -  **Train a Synthetic Data Generator** (updated)
       1. UI: variable selectors added
       2. UI: nominal input variables added
       3. ...

Reply:
I believe I have come to a workable compromise.  The lettered list seems to be giving me trouble, and so I have gone ahead with Roman number-based.  Tabs are now replaced with spaces.  This seems like some one-off problem where some other reason's making things look ugly.